README.md

```diff
@@ -4,15 +4,64 @@ Source code and documentation for the CAN Logger 2
 ## Purpose: 
 Use as an educational tool to interface with CAN, including dual CAN channels, MCP2515 third CAN, Single Wire CAN, LIN, J1708, WIFI, LEDs, and pushed button
 
+## Submodules
+The CAN Logger 2 depends on the following libraries:
+  * FlexCAN_Library forked from Collin80: https://github.com/Heavy-Vehicle-Networking-At-U-Tulsa/FlexCAN_Library
+  * MCP_CAN_lib forked from Cory Fowler: https://github.com/Heavy-Vehicle-Networking-At-U-Tulsa/MCP_CAN_lib
+  * Arduino's Wifi101 library: https://github.com/arduino-libraries/WiFi101
+  
+To use these submodules, you have to open a Git Bash command perform an update. Assuming Windows with Git installed:
```

Comment:
Do you suggest making the other library dependencies as submodules?

Reply:
I mention it here not because I think they should or should not be submodules, but rather because 1) the documentation (README.md) states that they are all submodules and 2) you mentioned the Duy was having problems building on his PC.

I think it makes licensing distinctions clearer if they are submodules

I think it also makes pull requests simpler and cloning faster.

Reply:
I'm going to tag this as an issue and address it in another pull request.